### PR TITLE
start your teacher training landing page

### PR DESCRIPTION
**Pull request by jenhadfield-dfe**
### Trello card
https://trello.com/c/wgIFGONm/6215-support-post-offer-engagement-test

### Changes proposed in this pull request
add landing page to support post offer engagement

### CODE REVIEW

File: app/views/content/trainees/start-your-teacher-training/_content.html.erb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <p>You'll train to teach in at least 2 schools, usually for a minimum of 6 weeks each, so you'll have plenty of time to meet other teachers and become part of supportive school communities.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Developing pupils</h2>
+    <div class="inset">
+      <p>During training, you'll get to find out what it takes to become the kind of teacher that pupils will remember years after leaving school.</p>
+      <p>Your lessons will be unobserved and observed, giving you plenty of chances to see pupils experience 'aha' moments, when your teaching helps them learn something new.</p>
+      <p>You'll be given the tools needed to inspire, motivate and challenge pupils. You'll find out how to adapt your teaching style to suit the strengths and needs of all pupils, and how to manage behaviour when things go off script.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/trainees/start-your-teacher-training/_content.html.erb:15]:
```suggestion
      <p>You'll be given the tools needed to inspire, motivate and challenge pupils. You'll find out how to adapt your teaching style to suit the strengths and needs of all pupils, and how to manage behaviour and create a positive learning environment.</p>
```
@Sarah-DfE Not sure about the phrase `when things go off script`? Happy for you to change my suggestion though

File: app/views/content/trainees/start-your-teacher-training/_content.html.erb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <p>Ultimately, you'll get to see how your teaching can help pupils progress and get the outcomes you'll know they are capable of.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Developing you</h2>
+    <div class="inset">
+      <p>You'll learn from teachers like you through taught sessions, seminars and workshops. You'll have the chance to get involved in academic or professional research so you can dig deeper into what makes you tick as a teacher.</p>
+      <p>You'll be able to see how you're getting on by completing tasks and assignments, so you can track what you're learning as you're teaching. These will be a combination of less formal, ongoing assessments, and formal assessments that usually happen at the end of your course.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/trainees/start-your-teacher-training/_content.html.erb:24]: @Sarah-DfE this sentence feels a bit hard to read with the repeated `you're`?

```suggestion
      <p>You'll be able to see how you're getting on by completing tasks and assignments, so you can track what you're learning as you teach. These will be a combination of less formal, ongoing assessments, and formal assessments that usually happen at the end of your course.</p>
```

File: app/views/content/trainees/start-your-teacher-training/_content.html.erb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      <div class="inset">
+        <p>Your provider will be able to answer any questions you have about your training.</p>
+        <p>Providers can answer practical questions about your placement, including:</p>
+        <p><ul>
+        <li>what a typical training term is like</li>
+        <li>how much time is spent in school or on academic learning</li>
+        <li>how you'll be assessed with exams and coursework</li></ul>
+        </p>
+        <p>Feeling prepared is half the challenge, so if you have anything you feel unsure about, speak to your provider.</p>
+      </div>
```
Comment (gemmadallmandfe) [inline app/views/content/trainees/start-your-teacher-training/_content.html.erb:50]: @Sarah-DfE Should we end with more of a positive note - eg `Good luck with your training!` as per email or similar?

**Review by Sarah-DfE — APPROVED** [review]
@gemmadallmandfe this looks good to me

File: app/views/content/trainees/start-your-teacher-training/_content.html.erb [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        <p>Your provider will be able to answer any questions you have about your training.</p>
+        <p>Providers can answer practical questions about your placement, including:</p>
+        <p><ul>
+        <li>what a typical training term is like</li>
+        <li>how much time is spent in school or on academic learning</li>
+        <li>how you'll be assessed with exams and coursework</li></ul>
+        </p>
+        <p>Feeling prepared is half the challenge, so if you have anything you feel unsure about, speak to your provider.</p>
+        
+        Good luck with your training!
```
Comment (gemmadallmandfe) [inline app/views/content/trainees/start-your-teacher-training/_content.html.erb:51]: @Sarah-DfE Does this not need `<p>` tags around it?

File: app/views/content/trainees/start-your-teacher-training/_content.html.erb [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      <div class="inset">
+        <p>Your provider will be able to answer any questions you have about your training.</p>
+        <p>Providers can answer practical questions about your placement, including:</p>
+        <p><ul>
+        <li>what a typical training term is like</li>
+        <li>how much time is spent in school or on academic learning</li>
+        <li>how you'll be assessed with exams and coursework</li></ul>
+        </p>
+        <p>Feeling prepared is half the challenge, so if you have anything you feel unsure about, speak to your provider.</p>
+
```
Comment (gemmadallmandfe) [inline app/views/content/trainees/start-your-teacher-training/_content.html.erb:50]: @Sarah-DfE this is coming up as a linting error - think it needs removing?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Comment by gemmadallmandfe** [comment]
Reviewed by policy

**Pull request merged**